helpers.json

```diff
@@ -1,502 +1,574 @@
+            ]}
+          ]
+        },
+        {
+          "name": "h_canIBJ",
+          "requires": [
+            "canIBJ",
+            "h_canUseMorphBombs"
+          ]
+        },
+        {
+          "name": "h_canFly",
+          "requires": [
+            {"or": [
+              "h_canIBJ",
+              "SpaceJump"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canUseSpringBall",
+          "requires": [
+            "SpringBall",
+            "Morph"
+          ]
+        },
+        {
+          "name": "h_hasBeamUpgrade",
+          "requires": [
+            {"or": [
+              "Charge",
+              "Ice",
+              "Wave",
+              "Spazer",
+              "Plasma"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canBlueGateGlitch",
+          "requires": [
+            "canGateGlitch",
+            {"or": [
+              {"ammo": { "type": "Missile", "count": 1 }},
+              {"ammo": { "type": "Super", "count": 1 }}
+            ]},
+            "h_BlueGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canGreenGateGlitch",
+          "requires": [
+            "canGateGlitch",
+            {"ammo": { "type": "Super", "count": 1 }},
+            "h_GreenGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canHeatedBlueGateGlitch",
+          "requires": [ 
+            "h_canNavigateHeatRooms",
+            {"heatFrames": 60},
+            "canGateGlitch",
+            {"or": [
+              {"ammo": { "type": "Missile", "count": 1 }},
+              {"ammo": { "type": "Super", "count": 1 }}
+            ]},
+            "h_HeatedBlueGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canHeatedGreenGateGlitch",
+          "requires": [ 
+            "h_canNavigateHeatRooms",
+            {"heatFrames": 60},
+            "canGateGlitch",
+            {"ammo": { "type": "Super", "count": 1 }},
+            "h_HeatedGreenGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canCrouchJumpDownGrab",
+          "requires": [ 
+            "canCrouchJump",
+            "canDownGrab"
+          ]
+        },
+        {
+          "name": "h_canPlasmaHitbox",
+          "requires": [
+            "Plasma",
+            "canHitbox"
+          ]
+        },
+        {
+          "name": "h_canTwoTileJumpMorph",
+          "requires": [
+            {"or": [
+              "can2HighWallMidAirMorph",
+              "canRJump"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canThreeTileJumpMorph",
+          "requires": [
+            {"or": [
+              "h_canTwoTileJumpMorph",
+              "can3HighMidAirMorph"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canOpenZebetites",
+          "requires": [
+            {"or": [
+              {"ammo": { "type": "Missile", "count": 12 }},
```

Comment:
This is 11 plus 1 leniency.  
I think I'd like to add a `or:[canDodgeWhileShooting, more_leniency]` too
Not that it really matters.

Reply:
I partially fixed it. I'm not sure how reliably you can open them with 11 missiles, if so, I could add an extra leniency helper that only is used when using only missles.

Reply:
Looks like it heals every 2 seconds, with the first tick being instant.  Two rinka/turret hits would be enough delay, or getting hit while jumping a gap.  To bring the ammo costs to what is used here.  That makes it a tighter window when using more missiles.  Also when using more missiles it is more likely to waste one in overkill.  
Should be alright as it is.